### ISSUE TITLE
Batiscaf + Jupyter

### ISSUE THREAD

**Issue by ten0s** [vladimir-vg/batiscaph#15]
Just an idea, what if integrate Batiscaf into Jupyter notebook using projects like https://github.com/filmor/ierl or https://github.com/robbielynch/ierlang. You automatically get the basic shell, history, ability to share notebooks, etc.

Pretty cool project!

**Comment by vladimir-vg** [comment]
Yeah, the idea of sharing ran scenarios with others is quite awesome. I'm hoping to add something like that in future, when Batiscaph would be able to work as a service.

This Jupiter thing in some ways looks quite similar to Batiscaph. I don't know how exactly it could be integrated, though.
